app/js/enhancedDate.js

```diff
@@ -1,5 +1,156 @@
-/* 
- * add your module definition for enhancedDate here
- * assign the module to a global variable
- * /
+var enhancedDate = (function (date) {
+// still maintains access to all globals
+  
+  var dateMethods = {};
+  var myDate = date;
+  var dateString = Date(year, month, day, hours, minutes, seconds, milliseconds);
+  var newDate = new Date();
+  var epochDate = newDate.getTime();
+  
+//Stores the passed-in date.  
+  dateMethods.checkDate = function () {
```

Comment:
Is this actually setDate? It looks like it would be. Your module should have a setDate method, so this method should be called setDate (if my assumption is correct.)